src/draw/sw/lv_draw_sw_vector.c

```diff
@@ -40,6 +43,10 @@ typedef struct {
     uint8_t a;
 } _tvg_color;
 
+typedef struct {
+    Tvg_Canvas * canvas;
```

Comment:
Nit: why this type does not follow the snake case style?

Reply:
It's coming from ThorVG, a 3rd party library.